File: 7-kyu/alphabetical-addition/python/solution.py
def add_letters(*letters):
    if not letters:
        return 'z'
    total = sum(ord(letter) - ord('a') + 1 for letter in letters)
    if total % 26 == 0:
        return 'z'
    return chr(total % 26 + ord('a') - 1)

File: 7-kyu/alphabetical-addition/javascript/solution.js
const addLetters = (...letters) => {
  if (letters.length === 0) {
    return 'z';
  }
  let total = letters.reduce((accum, letter) => {
    const charCode = letter.charCodeAt(0) - 'a'.charCodeAt(0) + 1;
    return accum + charCode;
  }, 0);
  if (total % 26 === 0) {
    return 'z';
  }
  return String.fromCharCode(total % 26 + 'a'.charCodeAt(0) - 1);
};
